src/sage/geometry/polyhedron/generating_function.py

```diff
@@ -122,18 +135,21 @@ def generating_function_of_integral_points(polyhedron, split=False,
 
     .. NOTE::
 
-        At the moment, only polyhedra with nonnegative coordinates
-        (i.e. a polyhedron in the nonnegative orthant) are handled.
+        At the moment, ``algorithm="omega"`` only handles polyhedra with
+        nonnegative coordinates (i.e., polyhedra in the nonnegative orthant).
+
+        ``algorithm="latte"`` option can handle general polyhedra,
+        but has no loggings for the decompositions and
+        only supports a tuple output unless the tuple has only one element.
```

Comment:
What does this mean?

Reply:
In the original implementation calling ``algorithm="omega"``, it has logging for the triangulations when split=True,
see line 461-462. But I don't know how to obtain the logging information for ``algorithm="latte"``.
The output setting means that for ``algorithm="omega"`, by default (``result_as_tuple=None``), the output format is automatically determined. While for ``algorithm="latte"``, it only support ``result_as_tuple=True``, see line 658-665.

How about the following changes:
``algorithm="latte"`` option can handle general polyhedra,
 but has no loggings for the triangulations when ``split=True`` and
 only supports a tuple output (``result_as_tuple=True``)
 unless the tuple has only one element.